keep/api/routes/incidents.py

```diff
@@ -327,49 +328,39 @@ def delete_alerts_from_incident(
 
     return Response(status_code=202)
 
-
 @router.post(
     "/mine",
     description="Create incidents using historical alerts",
 )
 def mine(
-    authenticated_entity: AuthenticatedEntity = Depends(AuthVerifier()),
-    use_n_historical_alerts: int = 10000,
-    incident_sliding_window_size: int = 6 * 24 * 60 * 60,
-    statistic_sliding_window_size: int = 60 * 60,
-    jaccard_threshold: float = 0.0,
-    fingerprint_threshold: int = 1,
+    authenticated_entity: AuthenticatedEntity = Depends(AuthVerifier(["read:alert"])),
+    alert_lower_timestamp: datetime = datetime.now() - timedelta(days=100),
```

Comment:
Here also better to move calculation out of function arguments

Reply:
Thx! Addressed [here](https://github.com/keephq/keep/commit/76508eb1a91805d71e26669945b05157dbb96486)